clients/admin-ui/cypress/e2e/datasets-classify.cy.ts

```diff
@@ -88,12 +88,6 @@ describe("Datasets with Fides Classify", () => {
       cy.wait("@getDatasets");
 
       cy.url().should("match", /dataset$/);
-
-      // The combination of Next routing and a toast message makes Cypress get weird
-      // when re-running this test case. Introducing a delay fixes it.
-      // eslint-disable-next-line cypress/no-unnecessary-waiting
-      cy.wait(100);
-      cy.getByTestId("toast-success-msg");
```

Comment:
hmm what happened to the toast?

Reply:
Since it looks like the toasts are broken [only in the dev environment](https://github.com/ethyca/fides/issues/2268#issuecomment-1387449219), let's restore these tests and make sure they pass in CI (or instead of `npm run dev`, use `npm run cy:start` and then `npm run cy:run` to make sure things pass that way)